.metadata/.plugins/org.eclipse.core.resources/.history/13/c0ad574346f300181af6d1b37b9a2943

```diff
@@ -0,0 +1,90 @@
+package com.herokuapp;
+
+import org.openqa.selenium.By;
+import data.function.CheckAssertion;
+
+public class PageElements extends PageFunctions{
```

Comment:
what is a page element and what is a page function, please stick to the naming convention as our original framework